Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/armory.yml

```diff
@@ -0,0 +1,82 @@
+- type: entity
+  id: CratePV3Pistol
+  parent: CrateWeaponSecure
+  name: PV-3 Mini Crate
+  description: "A crate containing three PV-3 Mini energy Personal Defense Weapons"
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponLaserPv3Mini
+      - id: WeaponLaserPv3Mini
+      - id: WeaponLaserPv3Mini
+
+- type: entity
+  id: CratePV3Smg
+  parent: CrateWeaponSecure
+  name: PV-3 Compact Crate
+  description: "A crate containing three PV-3 Compact Energy Assault Weapons"
```

Comment:
Just since they're the Compact SMG versions of the weapons, and not the (to be done?) full size Rifle versions, perhaps assault isn't the best choice of words?